src/ctapipe/monitoring/aggregator.py

```diff
@@ -167,16 +169,27 @@ def compute_stats(self, images, masked_pixels_of_sample) -> StatisticsContainer:
         masked_images = np.ma.array(images, mask=masked_pixels_of_sample)
 
         # Compute the mean, median, and std over the chunk per pixel
-        pixel_mean, pixel_median, pixel_std = sigma_clipped_stats(
+        # Use sigma_clip to get the clipped data, then compute stats from it
+        from astropy.stats import sigma_clip
```

Comment:
Why move the import?

Reply:
we only use it here, but not really important in this context. Will move up.